brightlocal/phpwhois/CVE-2015-5243.yaml

```diff
@@ -0,0 +1,8 @@
+title:     PHP Code Injection
+link:      https://github.com/sbaresearch/advisories/tree/public/2018/SBA-ADV-20180425-01_phpWhois_Code_Execution
+cve:       CVE-2015-5243
+branches:
+    master:
+        time:       null
+        versions:   ['<=4.2.5']
+reference: composer://brightlocal/phpwhois
```

Comment:
Why are forks included in the advisory?

Reply:
well, if these projects are registered on packagist under their own names, it totally makes sense to mark them as vulnerable.

Reply:
They are all registered on packagist. I determined vulnerable projects by searching for phpwhois on the packagist.org and then manually inspecting each project, if they contain the vulnerable lines of code.